package.json

```diff
@@ -65,20 +66,21 @@
     "zlib": "^1.0.5"
   },
   "scripts": {
-    "build": "NODE_ENV=production npm-run-all -s build:css build:js && cp ./_headers.${CONTEXT:-production} ./build/_headers",
+    "build": "next build",
     "build:css": "postcss src/tailwind.css -o src/index.css",
-    "build:js": "react-scripts build",
+    "build:next": "next build",
     "commit": "git-cz",
     "commit:retry": "npm run commit -- --retry",
+    "dev": "next dev",
```

Comment:
this run postcss automatically ?

Reply:
yup, i'm also trying out the tailwind `just-in-time` mode but there seem to be some custom styles in our index.css that are missing

Reply:
ok nevermind it is breaking things and the speed up is only for dev